### ISSUE TITLE
Silently fails on linux when you have insufficient privs

### ISSUE THREAD

**Issue by wez** [arcnmx/ddcset-rs#4]
I think it would be good if `ddcset` had a non-zero exit status and printed out something like this in that case:

```
No displays were found, and there was at least one i2c bus that you
lack permission to access.  Try running again as root?
```

(Unlike the equivalent issue I filed for `monitor-switch` https://github.com/mjkoo/monitor-switch/issues/1, I don't think it is safe to suggest installing `ddcset` with the setuid bit set)

**Comment by arcnmx** [comment]
Oh nice, someone else is using this stuff! Agreed, at the very least trying to run an operation while 0 displays are detected shouldn't exit successfully. Longer-term goal would probably be to not just [filter out displays that fail to enumerate](https://github.com/arcnmx/ddc-hi-rs/blob/ae5c655a6c46266a5db1fbe77774c4bf6b525c21/src/lib.rs#L376) if we can manage to have some degree of confidence that a random i2c bus is indeed a display.

> I don't think it is safe to suggest installing ddcset with the setuid bit set

Yeah probably not the best idea, a better suggestion might be to set up group permissions via udev and/or sudo. Perhaps ideally DRM should have a way to provide graphical users access to operations like this but I guess that's messy and that subsystem cares about little more than basic EDID/modes wrt i2c anyway.

**Comment by arcnmx** [comment]
the experience isn't quite that friendly, but this was resolved as part of 8763e1e458943fce2df6ef98d9fbf07df5c0c476
